dataactvalidator/config/sqlrules/d30_detached_award_financial_assistance.sql

```diff
@@ -0,0 +1,8 @@
+-- BusinessFundsIndicator must contain one of the following values: REC or NON.
+SELECT
+    row_number,
+    business_funds_indicator
+FROM detached_award_financial_assistance
+WHERE submission_id = {0}
+    AND UPPER(business_funds_indicator) != 'REC'
```

Comment:
It's a little cleaner if we combine these checks into one with something like `UPPER(business_funds_indicator) NOT IN ('REC', 'NON')`